### PR TITLE
Get Group notification update

### PR DESCRIPTION
**Pull request by dragosgaftoneanu**
## Description:
**What's changed?** 
- added note that the request to /api/v1/groups/{{groupId}} is supported only server side as confirmed by @AshishVaidya-okta on OKTA-226281
- corrected minor grammar in List Groups note

**Is this PR related to a Monolith release?**
No

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/groups/index.md [hunk 1]
```diff
@@ -110,6 +110,8 @@ Fetches a specific group by `id` from your organization
 
 Fetched [Group](#group-model)
 
+> This request is not supported from a browser and should be called server side.
```
Comment (jakubvul) [inline packages/@okta/vuepress-site/docs/reference/api/groups/index.md]: I'm not clear on what this means.

Comment (AshishVaidya-okta) [reply packages/@okta/vuepress-site/docs/reference/api/groups/index.md]: I think better wording would be:
`This request is not supported from a browser and should not be called from there.`

If we're willing to share the technical details, which I suspect devs would find out on their own anyway, then we should say something along the lines of:
`The API returns a JSON response and if a browser doesn't mention that in the 'Accept Headers' then the API shouldn't be called from there`

Comment (dragosgaftoneanu) [reply packages/@okta/vuepress-site/docs/reference/api/groups/index.md]: Hi @jakubvul @AshishVaidya-okta 

Modified the note to `This request is not supported from a browser and should not be called from there.`.

Comment (dragosgaftoneanu) [reply packages/@okta/vuepress-site/docs/reference/api/groups/index.md]: Hi @jakubvul @AshishVaidya-okta 

Any updates for this PR?

Comment (jakubvul) [reply packages/@okta/vuepress-site/docs/reference/api/groups/index.md]: So is the problem essentially that you must include the `Accept: application/json` header in your request?

Comment (AshishVaidya-okta) [reply packages/@okta/vuepress-site/docs/reference/api/groups/index.md]: Yes, that's my understanding following the comment from Brian Z. in the ticket. And that it happens for all our APIs. Are there reports of devs running into same problem for other APIs as the one here? Ideally, they should if browsers don't include `application/json` in the accept header. My understanding is that browsers don't request json by default -- https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation/List_of_default_Accept_values.

So if devs can somehow add that to the accept header while calling from browsers then things will work, just as they do when hitting our API thru Postman. We still need to fix the error response though. It should be 406, and not 5xx.

Comment (jakubvul) [reply packages/@okta/vuepress-site/docs/reference/api/groups/index.md]: We tell people that they must include that header in our API overview page: https://developer.okta.com/docs/reference/api-overview/#media-types

It is also included in every sample request, including the one for this call.

Comment (dragosgaftoneanu) [reply packages/@okta/vuepress-site/docs/reference/api/groups/index.md]: @jakubvul Removed the initial note and left only the grammar update.

**Pull request merged**